pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[tool.ruff]
+# Enable the pycodestyle (`E`) and Pyflakes (`F`) rules by default.
+# Unlike Flake8, Ruff doesn't enable pycodestyle warnings (`W`) or
+# McCabe complexity (`C901`) by default.
+select = [
+    # Pyflakes
+    "F",
+    # Pycodestyle
+    "E",
+    # isort
+    "I001"
+]
+ignore = [
+    "E501",
+    "E402", # ignores: Module level import not at top of file.  (isort takes care of this and if not, there is usually a good reason)
```

Comment:
I'm a little confused about `isort` - do we still plan to use it, if ruff also supports organizing imports?

Reply:
That would seem superfluous and would potentially just complicate things if the tools had slightly differing opinions about import placement.  I say we just use ruff and archive the isort ticket.